plugins/fix_path/__init__.py

```diff
@@ -0,0 +1,12 @@
+def build_filter(args):
+    return Filter(args)
+
+class Filter:
+    def __init__(self, args):
+        i = 1
+
+    # file_data = {'filename':filename,'file_ctx':file_ctx,'d':d}
+    def file_data_filter(self, file_data):
+        filename = file_data['filename']
+        if filename and '//' in filename:
```

Comment:
Why do this conditionally? The filename cannot be empty. Also, why scan the string twice?

Reply:
Forget the comment about compiling the regexp, I read the code hastily, sorry.